packages/auth/amplify_auth_cognito_test/README.md

```diff
@@ -0,0 +1,11 @@
+Tests for the [`amplify_auth_cognito_dart`](../amplify_auth_cognito_dart) package separated out so that they can be easily tested using `build_runner`.
+
+## Testing
+
+To run tests:
+
+```sh
+$ dart test
+$ dart run build_runner test --delete-conflicting-outputs -- -p chrome,firefox
+$ dart run build_runner test --release --delete-conflicting-outputs -- -p chrome,firefox
```

Comment:
Can this work with Edge, like Flutter tests? If so, might be worth noting somewhere.

Reply:
Oh yeah, I forget about edge 😂 I'll see if we can get it running on windows

Reply:
Looks like it's not supported yet, unfortunately: https://github.com/dart-lang/test/issues/1142. The Flutter team jumped through some hoops to get it to work (https://github.com/flutter/engine/pull/15641) and I'm not sure if that effort is worth it right now.

Reply:
Ok, sounds good